docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc

```diff
@@ -235,71 +241,55 @@ While `quarkus-oidc` and `quarkus-elytron-security-oauth2` can also verify such
 
 [NOTE]
 ====
-Your decision to choose whether to use opaque or JWT token format will be driven by architectural considerations.
+Architectural considerations drive your decision to use opaque or JWT token format.
 Opaque tokens tend to be much shorter than JWT tokens but need most of the token-associated state to be maintained in the provider database.
 Opaque tokens are effectively database pointers.
-JWT tokens are significantly longer than the opaque tokens but the providers are effectively delegating storing most of the token-associated state to the client by storing it as the token claims and either signing or encrypting them.
+JWT tokens are significantly longer than opaque tokens. Still, the providers effectively delegate most of the token-associated state to the client by storing it as the token claims and either signing or encrypting them.
 ====
 
-The following table provides a summary of the options for each authentication mechanism:
+.Credential types and authentication mechanisms
+|===
+^|Credential type ^| Authentication ^| Notes
+
+^|Username and password ^|xref:security-basic-authentication-concept.adoc[Basic ], xref:security-basic-authentication-concept.adoc#form-auth[Form], xref:security-oidc-bearer-authentication-concept.adoc[OIDC Bearer] ^|
+
+^|Bearer Access Token  ^| xref:security-oidc-bearer-authentication-concept.adoc[OIDC Bearer], xref:security-jwt.adoc[JWT], xref:security-oauth2.adoc[OAuth2]  ^|See the <<table>> table below.
+
+^|Single Sign-On (SSO) ^|xref:security-oidc-bearer-authentication-concept.adoc[OIDC Bearer], xref:security-basic-authentication-concept.adoc#form-auth[Form]   ^|
```

Comment:
It is a built-in authentication where the form auth is described

Reply:
So, do you want me to remove the SSO row completly, or replace SSO with the Built-in Aut?
@sberyozkin

Reply:
@MichalMaler No, I meant that the form link is wrong, points to the basic auth

Reply:
Fixing. My bad.